spec/requests/oauth/sessions_controller_spec.rb

```diff
@@ -219,15 +219,45 @@
     let_it_be(:authorization_grant) { create(:authorization_grant, user:) }
     let(:params) { { grant_type: } }
     let(:grant_type) { 'urn:ietf:params:oauth:grant-type:token-exchange' }
+    let(:subject_token) { 'subject-token' }
+    let(:resource) { 'resource' }
+    let(:subject_token_type) { 'subject-token-type' }
+    let(:client_id) { 'client-id' }
 
     include_context 'with an authenticated client', :post, :oauth_create_session_path
 
     it_behaves_like 'an endpoint that requires client authentication'
 
-    it 'creates an OAuth session and serializes the token data' do
-      call_endpoint
-      expect(response).to have_http_status(:ok)
+    # it 'creates an OAuth session and serializes the token data' do
+    #   call_endpoint
+    #   expect(response).to have_http_status(:ok)
+      ### need to create expected endpoint tests
```

Comment:
@dickdavis I may add another test here to test endpoint response or include that down the road in another PR that tests the type of `resource` or  type of `subject_token_type`